Resources/Prototypes/Catalog/Fills/Crates/armory.yml

```diff
@@ -28,7 +28,7 @@
   id: CrateTrackingImplants
   parent: [ CrateWeaponSecure, BaseRestrictedContraband ]
   name: tracking implants
-  description: Contains a handful of tracking implanters. Good for prisoners you'd like to release but still keep track of.
+  description: Contains five tracking implanters. Good for prisoners you'd like to release but still keep track of.
```

Comment:
i think the vagueness of the old description is fine and a bit less dry

Reply:
Gotcha; yeah, I was just swapping out other numbers (2 for two) and caught that. Swapping it back is no problem at all